cro-clib/src/types.rs

```diff
@@ -0,0 +1,67 @@
+use client_common::{PrivateKey, PublicKey};
+use client_core::HDSeed;
+use std::ffi::CStr;
+use std::os::raw::c_char;
+use std::os::raw::c_int;
+pub type CroHDWalletPtr = *mut CroHDWallet;
+pub type CroAddressPtr = *mut CroAddress;
+
+pub const SUCCESS: i32 = 0;
+pub const FAIL: i32 = -1;
+
+/// network kinds
+#[repr(C)]
+pub enum CroNetwork {
+    Mainnet,
+    Testnet,
+    Devnet,
+}
```

Comment:
This is identical to `chain_core::init::network::Network;`

-- perhaps just add `#[repr(C)]` on `chain_core::init::network::Network` and here `pub use chain_core::init::network::Network;`

Reply:
ok